imports/ui/QuestionEditItem.jsx

```diff
@@ -481,12 +468,9 @@ export class QuestionEditItem extends Component {
     this.setState({ showShareModal: !this.state.showShareModal })
   }
 
-  shareQuestion (user, approved) {
-    let question = this.state.question
-    question.shared = true
-    this.setState({ question: question }, () => {
-      this.duplicateQuestion(user, approved)
-    })
+  shareQuestion (user) {  
+    let question = _.extend({}, this.state.question)
+    Meteor.call('questions.share', question, user._id)
```

Comment:
share by email, not user id, so that students can never determine another's student id programatically.